packages/daemon/src/host.js

```diff
@@ -170,7 +171,7 @@ export const makeHostMaker = ({
      * @param {string | 'NONE' | 'SELF' | 'ENDO'} partyName
      */
     const providePowersFormulaIdentifier = async partyName => {
-      let guestFormulaIdentifier = lookupFormulaIdentifierForName(partyName);
+      let guestFormulaIdentifier = identifyLocal(partyName);
```

Comment:
I like abbreviating `lookupFormulaIdentifierForName` to `identifyLocal` generally. That could be one easy-to-review mechanical refactor on its own.

Since the difference between `identify` and `identifyLocal` is more like `identifyPetNamePath` vs `identifyPetName` respectively, I could buy `identify` (implied pet name path since it’s public API) and `identifyName` instead.